libs/vm_modules/examples/commodity_prediction/script/commodity_prediction.etch

```diff
@@ -52,18 +52,37 @@ if (System.Argc() != 3)
   return;
 endif
 
-var loader = CommodityDataLoader();
-loader.AddData(System.Argv(1), System.Argv(2));
+var loader = DataLoader();
+loader.AddData("commodity", System.Argv(1), System.Argv(2));
 
 var graph = Graph();
 build_graph(graph);
-read_weights(graph);
 
-while(!loader.IsDone())
-    var input_data = loader.GetNext();
-    graph.SetInput("num_input", input_data.Data());
-    var pred = graph.Evaluate("softmax_3");
-    print(pred.ToString());
-endwhile
+var testing = false;
```

Comment:
ideally this would be an input argument

Reply:
I can't figure out how to convert a string to a bool in etch, so this will have to wait for a future PR!